app/src/main/res/xml/appfilter.xml

```diff
@@ -1795,6 +1804,8 @@
     <!-- Microsoft Excel -->
     <item component="ComponentInfo{com.microsoft.office.excel/com.microsoft.office.apphost.LaunchActivity}" drawable="excel" />
 
+    <!-- Microsoft Launcher -->
+    <item component="ComponentInfo{com.microsoft.launcher/com.microsoft.launcher.Launcher}" drawable="microsoftlauncher"/>
```

Comment:
Could you possibly add a newline after this entry?

Reply:
Both problems fixed